packages/test/test-end-to-end-tests/src/test/gc/gcDatastoreAliased.spec.ts

```diff
@@ -110,8 +105,17 @@ describeNoCompat("GC Data Store Aliased", (getTestObjectProvider) => {
         // Should be able to retrieve root datastore from remote
         const containerRuntime2 = mainDataStore2.containerRuntime;
         assert.doesNotThrow(async () => containerRuntime2.getRootDataStore(alias), "Aliased datastore should be root!");
+        const aliasableDataStore2 = await containerRuntime2.getRootDataStore(alias);
+        const aliasedDataStoreResponse2 = await aliasableDataStore2.request({url:"/"});
+        const aliasedDataStore2 = aliasedDataStoreResponse2.value as TestDataObject;
 
-        await summarizeOnContainer(container2);
-        // TODO: Check GC is notified
+        const gcReferences =
+            (containerRuntime2 as any).garbageCollector.newReferencesSinceLastRun as Map<string,string[]>;
+        const references = gcReferences.get("/") as string[];
+        assert(references !== undefined, "Should be able to get the root datastore");
+        const reference = references.filter((ref) => ref === aliasedDataStore2.handle.absolutePath)[0];
+
+        assert.doesNotThrow(async () => summarizeOnContainer(container2), `Should be able to summarize!`);
```

Comment:
When would you expect the summarize to throw? I am not sure this check is doing anything.